<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ajout</class>
 <widget class="QDialog" name="Dialog_ajout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>System</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QGroupBox{
	background-color: rgb(255, 255, 255);
	/*background-color: rgb(255, 170, 0);*/

}

QLabel{
font-size :14px;
font-weight:bold;
font-family: 'Source Code Pro', monospace;
}
QLineEdit{ background-color:#ffffff;

border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
}

</string>
   </property>
   <property name="title">
    <string>Ajout Reservation</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>201</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed {
display: inline-block;
  padding: 20px 14px;
 
  cursor: pointer;
font-size: 11px;
outline: none;
  color: #fff;
  background-color: #4CAF50;
  border: none;
 text-align: center;
  text-decoration: none;
  box-shadow: 0 9px #999;
}
QPushButton:hover {background-color: #3e8e41}
QPushButton:active {
  background-color: #3e8e41;
  box-shadow: 0 5px #666;
  transform: translateY(4px);
}</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>201</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed {
display: inline-block;
  padding: 20px 14px;
 
  cursor: pointer;
font-size: 11px;
outline: none;
  color: #fff;
  background-color: #4CAF50;
  border: none;
 text-align: center;
  text-decoration: none;
  box-shadow: 0 9px #999;
}
QPushButton:hover {background-color: #3e8e41}
QPushButton:active {
  background-color: #3e8e41;
  box-shadow: 0 5px #666;
  transform: translateY(4px);
}</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_from_ajout">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>201</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed {
display: inline-block;
  padding: 20px 14px;
 
  cursor: pointer;
font-size: 11px;
outline: none;
  color: #fff;
  background-color: #4CAF50;
  border: none;
 text-align: center;
  text-decoration: none;
  box-shadow: 0 9px #999;
}
QPushButton:hover {background-color: #3e8e41}
QPushButton:active {
  background-color: #3e8e41;
  box-shadow: 0 5px #666;
  transform: translateY(4px);
}</string>
    </property>
    <property name="text">
     <string>undo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>88</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Reservation</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nomfilm_ajout">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>140</y>
      <width>35</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox:focus{
	background-color: rgb(255, 255, 127);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_reservation_ajout">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>88</y>
      <width>126</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:focus {background-color: rgb(255, 255, 127); }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>62</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Personne</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>164</y>
      <width>32</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_ajout">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>164</y>
      <width>110</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateTimeEdit:focus {background-color: rgb(255, 255, 127); }
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_client_ajout">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>62</y>
      <width>126</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:focus {background-color: rgb(255, 255, 127); }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>140</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom Film</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nb_personne_ajout">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>114</y>
      <width>126</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QNb_personne_ajout{ background-color:#ffffff;

border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;}
QLineEdit:focus {background-color: rgb(255, 255, 127); }</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
